lib/tasks/case_contact_types_reminder.rb

```diff
@@ -0,0 +1,99 @@
+class CaseContactTypesReminder
+  include Rails.application.routes.url_helpers
```

Comment:
what is this used for?

Reply:
removed. that was accidentally left in from testing dynamic url stuff